tensorflow/python/data/experimental/ops/io.py

```diff
@@ -38,7 +39,8 @@
     "tensorflow.python.saved_model.nested_structure_coder")
 
 
-@tf_export("data.experimental.save", v1=[])
+@tf_export("data.experimental.save", "data.save", v1=[])
```

Comment:
similarly to the other promoted methods, this should be promoted to non-experimental as `tf.data.Dataset.save()` so that it can be used as:

```
dataset = ... 
dataset.save()
```

instead of:

```
dataset = ...
tf.data.save(dataset)
```

Reply:
@jsimsa wouldn't this make the usage of `load` out of sync with `save` ?

Reply:
okay just saw your comment. Will make the changes.